<launch>
    <arg name="verbose" default="false"/>

    <node name="lee_position_controller_node" pkg="rotors_control" type="lee_position_controller_node" output="screen">
        <rosparam command="load" file="$(find rotors_gazebo)/resource/lee_controller_$(arg mav_name).yaml" />
        <rosparam command="load" file="$(find rotors_gazebo)/resource/$(arg mav_name).yaml" />
        <remap from="odometry" to="odometry_sensor1/odometry" />
    </node>


    <!--- Trajectory planner -->
    <node name="planner" pkg="mav_trajectory_generation_example" type="trajectory_generation_example" output="screen">
        <rosparam command="load" file="$(find mav_trajectory_generation_example)/cfg/$(arg mav_name)_params.yaml"/>
        <remap from="uav_pose" to="odometry_sensor1/odometry"/>
    </node>

    <!--- Trajectory sampler -->
    <node name="sampler" pkg="mav_trajectory_generation_ros" type="trajectory_sampler_node" output="screen">
        <remap from="path_segments_4D" to="trajectory"/>
    </node>


</launch>
